src/sota_tools/garage_deploy.cc

```diff
@@ -122,6 +122,14 @@ int main(int argc, char **argv) {
     LOG_FATAL << e.what();
     return EXIT_FAILURE;
   }
+
+  std::string cmd("garage check");
```

Comment:
It seems to have a typo, `garage-check` is the correct tool name.

But also, it would be best to not launch another process and just call some C++ code as garage-check's code lives in this repo. This way, garage-check wouldn't need to be in the `$PATH`.

It would probably require a small refactor to move garage-check's logic in a small library that would be used by `garage-check` and `garage-deploy`

Reply:
Seems a bit complicated than I thought, but this looks like the right way to do it. Thank you!